vm/packer/templates/general-purpose-cloud-vm/packer.in.json

```diff
@@ -0,0 +1,7 @@
+{
+  "license": "general-purpose-cloud-vm",
```

Comment:
Is is going to be Click to Deploy solution? If so, the licence should start with `c2d-` prefix, so `c2d-general-purpose-cloud-vm`.

Reply:
done